multi_gpu/run_parallel.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+
+# installation steps for Miniconda
+export HOME=$PWD
+sh Miniconda3-latest-Linux-x86_64.sh -b -p $HOME/miniconda3
+rm Miniconda3-latest-Linux-x86_64.sh
+export PATH=$HOME/miniconda3/bin:$PATH
+source $HOME/miniconda3/etc/profile.d/conda.sh
+hash -r
+conda config --set always_yes yes --set changeps1 no
+
+# install environment
+conda env create -f env.yml -n multigpu
+conda activate multigpu
+
+#wandb login <your api key>
+
+# Edit the argument after -n to reflect how many GPUs were requested in the submit file
+python3 model_parallel.py -n 1
```

Comment:
We can set this argument automatically.

@ChristinaLK what would be a good way for the executable script to refer back to the number of GPUs set in line 21 of the submit file? I can think of various ways to get the number of GPUs this job was assigned or was supposed to be assigned.  Which do you recommend?

Reply:
You could pass `$(request_gpus)` as an argument to the shell script, or pull it out of the `.machine.ad` file that's in the working directory - there should be an entry with the number of GPUs.